### ISSUE TITLE
Ignoring living_street in Car routing

### ISSUE THREAD

**Issue by fx99-D** [abrensch/brouter#296]
In the car fast profile, it looks as if residential and living_street were combined to a common class **isresidentialorliving** with a maxspeed_implicit of 30 (vs. legal 7 for living_street in Germany).
This results in a completely unreasonable car routing through a long living street using the BRouter web client.
For bike routing the difference is not that severe but still significant.

Is there a way to fix this ?

**Comment by polyscias** [comment]
Yes, it is quite easy to fix this, you can edit the profile and change the 30 into 7. In brouter-weg you can use the wrench button on the right side and select the profile tab and change and line 127 the value of 30 to 7.

Good to know there is a 7 km/h limit for Germany for living streets, I see there is a more complete overview on [Tag:highway=living_street](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dliving_street).

I think the profile should be changed to separate residential streets and living_street's. For residential streets an implicit limit of 30 km/h makes sense but as the data on the living_street page shows, 20 seems to be a better global implicit limit for living streets.
